wp-content/themes/borderzine/functions.php

```diff
@@ -12,6 +12,7 @@
 	'/inc/byline_class.php',
 	'/inc/compat-bunyad.php',
 	'/inc/post-tags.php',
+	'/inc/widgets/class-borderzine-3-col-widget.php',
```

Comment:
[Largo's widgets.php](https://github.com/INN/largo/blob/trunk/inc/widgets.php) does a lot more than just checking to see if a widget is enabled and then `require`ing the widget files and registering them. I think removing the widget `require`s in this theme to a file which is then `require`d by `functions.php` just adds steps of confusion for future maintainers, and it's probably better to keep the always-on `require`s in one place.

Reply:
@benlk Should we at least register all of the widgets in one spot like we did for Workday Minn? https://github.com/INN/umbrella-workdayminn/blob/70a90c7035e252034cd221c85ca985a33a88d88a/wp-content/themes/workday/functions.php#L85-L100

Reply:
I don't have a strong opinion there; in this PR I put the widget's registration in file containing the widget class so that if the file is ever removed, the widget registration would be, too. Goal is to prevent future errors.